.github/workflows/ci.yml

```diff
@@ -52,11 +52,14 @@ jobs:
         run: bin/test
 
       # test no uncommited changes
-      - name: test no uncommited changes
-        run: bin/test-no-uncommitted-doc-changes
-        env:
-          PYTHON_VERSION: ${{ matrix.python-version }}
-          PLONE_VERSION: ${{ matrix.plone-version }}
+      # TODO Temporarily commented out.
```

Comment:
@mauritsvanrees I am somehow reluctant to merge this PR because we comment out this check here. I know there are quirks but this works reliably for quite a while now. Can we try to fix things step by step? First the checkout, then the normalization?

Reply:
@tisto as the wrong version was being used, there were errors in the documentation that were "hidden". I think the best thing to do is to merge this PR with the documentation tests commented on and then fix the tests. Using the wrong version is a more serious mistake than not running the documentation tests

Reply:
> Can we try to fix things step by step? First the checkout, then the normalization?

No. If we only fix the checkout, we run into test failures for the part that checks if there are differences in the http-examples.

I have already fixed the documentation tests in PR https://github.com/plone/plone.restapi/pull/1215 which builds on the current PR.

Problem is that the follow-up PR changes 146 files, making it virtually impossible to review. That is why I split it into two PRs. But that is optional. You can also ignore the current PR and merge the other, as it includes all commits.